app/pages/project/classify.cjsx

```diff
@@ -321,6 +326,13 @@ module.exports = React.createClass
       apiClient.type('user_groups').get(props.location.query.group).then (group) =>
         @setState({ validUserGroup: group and group.links.users.includes(props.user.id) })
 
+  clearUserGroupForClassification: (props, context) ->
+    if (props.location.query?.group)
+      @setState({ validUserGroup: false })
+      newLocation = Object.assign({}, props.location, { query: '' });
```

Comment:
not relevant for initial use cases, but I think this would clear `workflow=1234` query param if present in project with experimental allow workflow query feature, not sure if more surgical way to clear just `group`.